CONTRIBUTING.md

```diff
@@ -5,20 +5,20 @@ First off, thanks for your interest in contributing!
 ## Reporting bugs
 
 We exclusively use [bugzilla](https://issues.dlang.org/) for issue tracking, which is why Github issues are disabled on this repository.
-If you found a bug, please [check bugzilla](https://issues.dlang.org/query.cgi) to see if it's already reported.
-If it isn't, you can [create a new issue](https://issues.dlang.org/enter_bug.cgi).
+Before reporting a bug, please [check bugzilla](https://issues.dlang.org/query.cgi) to see if it's already reported.
+If it isn't, [create a new issue](https://issues.dlang.org/enter_bug.cgi).
 
-If you have a question about a specific behavior, the [D.Learn](http://forum.dlang.org/group/learn) group is a good place to ask for clarification before reporting an issue.
+For questions about a specific behavior, the [D.Learn](http://forum.dlang.org/group/learn) group is a good place to ask for clarification before reporting an issue.
 
 ### Content
 
-When creating a new issue, make sure to include:
-- which version of DMD you are using (which can be found by running `dmd` with no argument).
+When creating a new issue, include:
+- the version of DMD being used (which can be found by running `dmd` with no argument).
```

Comment:
recommend `--version`?

Reply:
Simpler is better, and the version gets printed out if you just type dmd.

Reply:
fair enough.